util/flags.py

```diff
@@ -112,11 +112,26 @@ def create_flags():
     f.DEFINE_boolean('remove_export', False, 'whether to remove old exported models')
     f.DEFINE_boolean('export_tflite', False, 'export a graph ready for TF Lite engine')
     f.DEFINE_integer('n_steps', 16, 'how many timesteps to process at once by the export graph, higher values mean more latency')
-    f.DEFINE_string('export_language', '', 'language the model was trained on e.g. "en" or "English". Gets embedded into exported model.')
     f.DEFINE_boolean('export_zip', False, 'export a TFLite model and package with LM and info.json')
-    f.DEFINE_string('export_name', 'output_graph', 'name for the export model')
+    f.DEFINE_string('export_file_name', 'output_graph', 'name for the exported model file name')
     f.DEFINE_integer('export_beam_width', 500, 'default beam width to embed into exported graph')
 
+    # Model metadata
+
+    f.DEFINE_string('export_author_id', 'author', 'author of the exported model. This is a unique handle used for identifying and collating models per author. Suggestion: use your GitHub username or organization name')
+    f.DEFINE_string('export_model_name', 'model', 'name of the exported model')
```

Comment:
As this is used in the output file name there are likely constraints on the possible name used here, e.g. no forward slashes.